python/openmldb/sql_magic/sql_magic.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright 2021 4Paradigm
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import print_function
+from IPython.core.magic import (Magics, magics_class, line_magic,
+                                cell_magic, line_cell_magic)
+import openmldb
+import re
+@magics_class
+class SqlMagic(Magics):
+
+    def __init__(self, shell):
+        super(SqlMagic, self).__init__(shell)
+        db = openmldb.dbapi.connect('db_test','127.0.0.1:6181','/onebox')
```

Comment:
This should be passed by users since they may not `127.0.0.1:6181` and `/onebox` in their environment.

Reply:
updated